oat-sa/extension-tao-itemqti
# fix/AUT-2014/video-editing-size

**Related to:** [AUT-2014](https://oat-sa.atlassian.net/browse/AUT-2014)

**Description:**

When adding a video to an interaction, only the first time is calling to the Size editor on the widget panel.

**Changes:**

- Reset the player state to trigger the size editor every time is called.

**How to check:**

**Important: Disable cache must be unchecked on Developer Tools to be able to reproduce the original issue.**

- Add a Block interaction and add a video inside it.
- Click on the video to see its Size
- Click outside the video to close the Media properties
- Click again on the video and verify that the Size and deposition properties are loading.

**TAE**
https://oat-sa.atlassian.net/browse/AUT-2014?focusedCommentId=179094
